library/lib_window_vessel_stats.ks

```diff
@@ -0,0 +1,83 @@
+@lazyglobal off.
+
+run lib_window.
+run lib_navball.
+
+function open_window_vessel_stats{
+	parameter window.
+
+	local process_state to list(
+		list(false,window,"update_window_vessel_stats",false)
+	).
+	draw_window_vessel_stats(process_state).
+	return process_state.
+}
+
+function draw_window_vessel_stats{
+	parameter process_state.
+
+	if not is_process_gui(process_state){
+		return.
+	}
+
+	local window is get_process_window(process_state).
+
+	print "Vessel stats: (AG1 closes)" at(window[0]+2,window[1]+2).
+	print "Latitude: " at(window[0]+2,window[1]+4).
+	print "Longitude: " at(window[0]+2,window[1]+5).
+	print "Compass: " at(window[0]+2,window[1]+6).
+	print "Mass: " at(window[0]+2,window[1]+7).
+	print "Name: " at(window[0]+2,window[1]+8).
+	print "Time: " at(window[0]+2,window[1]+9).
+	print "SAS: " at(window[0]+2,window[1]+10).
+	print "RCS: " at(window[0]+2,window[1]+11).
+	print "GEAR: " at(window[0]+2,window[1]+12).
+	print "LIGHTS: " at(window[0]+2,window[1]+13).
+	print "PANELS: " at(window[0]+2,window[1]+14).
+}
+
+function update_window_vessel_stats{
+	parameter process_state.
+
+	if process_needs_redraw(process_state){
+		draw_window_vessel_stats(process_state).
+	}
+
+	local wnd is get_process_window(process_state).
+	if ag1{
+		end_process(process_state). //finished
+		return 0.
+	}
+	local sason is "OFF".
```

Comment:
It can be abstracted to something like this:
```
function bool_to_on_off {
  parameter bool.
  if bool {return "ON".}
  else {return "OFF".}
}
```
or even:
```
function trinary_operator {
  parameter
    bool,
    true_val,
    false_val.
  if bool {return true_val.}
  else {return false_val.}
}
```